stories/Dropdown.stories.ts

```diff
@@ -60,6 +60,7 @@ export const MultipleSelect = Template.bind({});
 MultipleSelect.args = {
   options: optionsWithMultiple,
   multiple: true,
+  clearButtonIsVisible: true,
```

Comment:
No, the multiple selection dropdown comes with options selected by default and therefore the button needs to be initialized as true.

Reply:
The setClearButtonIsVisible function is called in ngOnInit in case the dropdown is initialized with an option selected by default, if it has, the button is rendered, if not, it does not appear. There is a test to verify this, but in the storybook there are some bugs